src/tedi/components/content/accordion/accordion.module.scss

```
@@ -0,0 +1,295 @@
+      background: var(--general-surface-hover);
+    }
+
+    &:active {
+      background: var(--general-surface-brand-tertiary);
+    }
+
+    &:has(.tedi-accordion-item-header__trigger:focus-visible) {
+      z-index: 1;
+      background: var(--card-background-primary);
+      border: 1px solid var(--card-border-primary);
+      outline: none;
```

Stylelint (17.13.0)

[error] Expected "outline" to come before "border" (order/properties-order)

(order/properties-order)